### ISSUE TITLE
Calligra touch responsiveness is really bad on smartphone

### ISSUE THREAD

**Issue by llaske** [llaske/sugarizer#955]
On small screens like smartphone, touch responsiveness of Calligra activity is really bad.

![image](https://user-images.githubusercontent.com/981841/117584286-66f52d80-b10c-11eb-873c-55d9afe11d6c.png)

It's hard to use the activity.

**Comment by rahulkiller104** [comment]
@llaske can you explain this issue in more detail?

**Comment by llaske** [comment]
@rahulkiller104 It's related to precision (detection of cursor location). With fingers and when the screen is small it's hard to follow the letter. All the logic is [here](https://github.com/llaske/sugarizer/blob/dev/activities/Calligra.activity/js/player.js#L186). I think this code works for cursor/big screen but is not efficient for touch/small screen.

**Comment by rahulkiller104** [comment]
@llaske thanks for explaintion
